src/Serilog.Sinks.NLog/Sinks/NLog/NLogSink.cs

```diff
@@ -29,34 +29,49 @@ public void Emit(LogEvent logEvent)
                 }
             }
 
+            var level = GetMappedLevel(logEvent);
             var message = logEvent.RenderMessage(formatProvider);
             var exception = logEvent.Exception;
 
+            var nlogEvent = new LogEventInfo(level, loggerName, message)
+            {
+                Exception = exception
+            };
+
+            // pass along the event's properties to nlog
+            foreach (var property in logEvent.Properties)
+            {
+                // format simple scalar strings without wrapping quotes, which is more likely to be what nlog users expect
+                var format = (property.Value is ScalarValue && property.Value.GetType() == typeof(string))
```

Comment:
Should be more like:

```
var sv = property.Value as ScalarValue;
var format = sv != null && sv.Value is string ? "l" : null;
```

I.e. `property.Value.GetType()` will be returning `typeof(ScalarValue)` in the code as it stands. The value we need to check is a subproperty of `ScalarValue`, also called `Value`. Hope this helps :)